packages/website/components/account/filesManager/filesManager.js

```diff
@@ -25,13 +23,23 @@ import CheckIcon from 'assets/icons/check';
 
 const defaultQueryOrder = 'a-z';
 
-type FilesManagerProps = {
-  className?: string;
-  content?: any;
-  onFileUpload: () => void;
-};
+/**
+ * @typedef {import('web3.storage').Upload} Upload
+ */
+
+/**
+ * @typedef {Object} FilesManagerProps
+ * @prop {string} [className]
+ * @prop {any} [content]
+ * @prop {() => void} [onFileUpload]
```

Comment:
This was not optional above...?

Reply:
Yes good catch!! Will push a quick hotfix